### DISCUSSION TITLE
Will this extension still be maintained (for gnome) when Pop OS moves to its own DE ?

### DISCUSSION THREAD

**Discussion by rdong8** [pop-os/shell#1563] [Q&A]
Really love this extension but I'm on fedora linux

**Comment by dwedia** [comment]
I came here looking for an answer for this aswell.
I am also on Fedora (due to working in RedHat environment). This extension lets me work in gnome instead of i3 (Since I depend on Tiling / Tabbing heavily for my workflow).

**Comment by alexispurslane** [comment]
@rdong8 @Ramiraz80 Not a Pop dev, but I'm on here to ask a question of my own and figured I'd mention that I'm currently in the early stages of exploring how possible it would be to create a fresh version of this plugin designed for GNOME 43+. I'm going to probably be borrowing a lot of code from the Pop Shell to start with (why reinvent the wheel?) but the intention will to make it future-proof, a bit slimmed down, and not dependent on the System76 team. If either of you know JavaScript and are interested in helping lmk.
